### PR TITLE
add get started with kafka api

### PR DESCRIPTION
**Pull request by YangKian**

### CODE REVIEW

File: docs/start/get-started-with-kafka-api.md [hunk 1]
```diff
@@ -1 +1,150 @@
...
++------+------------+--------------------+
+| demo | 3          | 1                  |
++------+------------+--------------------+
+```
+
+## Produce data to a topic
+
+The `hstream-kafka produce` command can be used to produce data to a topic in a interactive shell.
+```sh
+hstream stream-kafka produce demo --separator "@" -i
```
Comment (daleiz) [inline docs/start/get-started-with-kafka-api.md]: command error

File: docs/start/get-started-with-kafka-api.md [hunk 2]
```diff
@@ -1 +1,150 @@
...
+```
+- With the `--separator` option, you can specify the separator for key. The default separator is "@". Using the separator, you can assign a key to each record. Record with same key will be append into same partition of the topic.
+- Using `-i` option to enter the interactive mode.
+
+```sh
+info@This is a info level log.
+warn@This is a warn level log.
+hello kafka!
```
Comment (daleiz) [inline docs/start/get-started-with-kafka-api.md]: hello, hstream!

**Review by daleiz — CHANGES_REQUESTED** [review]

**Comment by YangKian** [comment]
@g1eny0ung There is a markdown link error
> ERROR: 1 dead links found!
[✖] https://raw.githubusercontent.com/hstreamdb/docs-next/main/assets/kafka-quick-start.yaml → Status: 404

I think the reason is that this pr is not merged into main? The `kafka-quick-start.yaml` was introduced in this pr.

File: docs/start/get-started-with-kafka-api.md (hunk 1 above)
Comment (YangKian) [reply docs/start/get-started-with-kafka-api.md]: fixed

File: docs/start/get-started-with-kafka-api.md (hunk 2 above)
Comment (YangKian) [reply docs/start/get-started-with-kafka-api.md]: fixed

**Review by daleiz — APPROVED** [review]

**Pull request merged**